docs/guide/sessions.md

```diff
@@ -52,45 +52,56 @@ Symfony can be configured to use custom handlers, or just fall back to what is
+eZ Platform adapts Symfony's defaults to make sure it's session save path is always taken into account:
 
 ``` yaml
 # Default config.yml session configuration
 framework:
     session:
-        # handler_id set to null will use default session handler from php.ini
-        handler_id:  ~
+        # handler_id can be set to null (~) like default in Symfony, if so will use default session handler from php.ini
+        # But in order to use %ezplatform.session.save_path%, default eZ Platform instead sets %ezplatform.session.handler_id% to:
+        # - session.handler.native_file (default)
+        # - ezplatform.core.session.handler.native_redis (recommended value for Cluster usage, using php-redis session handler )
+        handler_id: '%ezplatform.session.handler_id%'
 ```
 
 ### Recommendations for production setup
 
 #### Single-server setup
 
-For a single server, the default handler is preferred.
+For a single server, the default file handler is preferred.
 
 #### Cluster setup
 
-For a [cluster](clustering.md) setup you need to configure sessions to use a back end that is shared between web servers and supports locking.
-The only options out of the box supporting this in Symfony are the native PHP Memcached session save handler
-provided by the `php-memcached` extension, and Symfony session handler for PDO (database).
+For a [cluster](clustering.md) setup you need to configure sessions to use a back end that is shared between web servers.
+The options out of the box in Symfony are the native PHP Memcached or Redis session handlers, and Symfony session handler for PDO _(database)_.
 
 ##### Storing sessions in Memcached using `php-memcached`
 
-To set up eZ Platform using Memcached you need to [configure the session save handler settings in `php.ini`](http://php.net/manual/en/memcached.sessions.php),
-and optionally tweak [`php-memcached` session settings](http://fr2.php.net/manual/en/memcached.configuration.php).
+To set up eZ Platform using [Memcached](https://pecl.php.net/package/memcached) you need to [configure the session save handler settings in `php.ini`](http://php.net/manual/en/memcached.sessions.php), optionally tweak [`php-memcached` session settings](http://php.net/manual/en/memcached.configuration.php), and set eZ Platform's `%ezplatform.session.handler_id%`
+to `~` _(null)_.
 
-##### Storing sessions in Redis using pecl package
+NOTE: For `php-memcached`, session locking should be kept enabled.
 
-To set up eZ Platform using the [Redis pecl package](https://pecl.php.net/package/redis)
-you need to [configure the session save handler settings in `php.ini`](https://github.com/phpredis/phpredis#php-session-handler).
+##### Storing sessions in Redis using `php-redis`
+
+To set up eZ Platform using the [Redis](https://pecl.php.net/package/redis)
+you need to [configure the session save handler settings in `php.ini`](https://github.com/phpredis/phpredis#php-session-handler),
```

Comment:
As far as I know, it's enough to configure it only on eZ (Symfony) side without touching `php.ini` configuration.

Reply:
You have an example here https://labs.madisoft.it/scaling-symfony-sessions-with-redis/  So yes you need to provide a DSN 
Configuring php.ini means that all your PHP project will use redis. It might not be something you want to do if you share many project on the same server. (or use php_value on vhost by example)

Reply:
Yeah updated doc show both ways now, both the way of using what php provides, and the way of using own handler. Both on memcached and redis. And on redis it's what we do by default on platform.sh.